contracts/internals/transferrable.sol

```diff
@@ -34,7 +34,8 @@ contract Transferrable {
     function _safeTransfer(address payable _to, address _asset, uint256 _amount) internal {
         // address(0) is used to denote ETH
         if (_asset == address(0)) {
-            _to.transfer(_amount);
+            (bool success, ) = _to.call.value(_amount)("");
```

Comment:
Looks good, though what's the reason for this change - gas limit?

Reply:
Yes, the problem is that due to the extra code added in the fallback function (delegate call etc.) ALL deposits made via
transfer() or send() are broken! At least it is good to offer users the possibility to send funds to other wallets without having to  use executeTransaction(). The problem still remains: if someone is trying to send funds through a contract (e.g. exchanges and they haven't updated they transfer function, the Tx will fail.